deploy/helm/sumologic/conf/logs/otelcol/config.yaml

```diff
@@ -557,8 +558,6 @@ service:
         - groupbyattrs/systemd
         - resource/add_cluster
         - source/kubelet
-        - transform/remove_attributes
```

Comment:
I wasn't sure why were removing the log attributes earlier

Reply:
We remove them for data from otel log collectors, because there the attributes come in as body attributes. So we delete existing record attributes and then copy attributes from the body to record attributes. Data coming in from Fluent Bit already has this in record attributes.